osbs/api.py

```diff
@@ -553,6 +558,29 @@ def create_worker_build(self, *args, **kwargs):
         return self.create_prod_build(*args, **kwargs)
 
     @osbsapi
+    def create_orchestrator_build(self, *args, **kwargs):
+        """
+        Create an orchestrator build
+
+        Pass through method to create_prod_build with the following
+        modifications:
+            - platforms param is required
+            - inner template set to orchestrator_inner.json if not set
+            - outer template set to orchestrator.json if not set
+            - customize configuration set to orchestrator_customize.json if not set
+
+        :return: BuildResponse instance
+        """
+        if not kwargs.get('platforms'):
+            raise ValueError('Orchestrator build requires platforms param')
+
+        kwargs.setdefault('inner_template', ORCHESTRATOR_INNER_TEMPLATE)
+        kwargs.setdefault('outer_template', ORCHESTRATOR_OUTER_TEMPLATE)
+        kwargs.setdefault('customize_conf', ORCHESTRATOR_CUSTOMIZE_CONF)
+
+        return self.create_prod_build(*args, **kwargs)
```

Comment:
This would return a single `BuildReponse`. Should koji-containerbuild handle it in some special way?